Userland/Libraries/LibWeb/Layout/Box.h

```diff
@@ -65,6 +69,7 @@ class Box : public NodeWithStyleAndBoxModelMetrics {
     Optional<CSSPixels> m_natural_height;
     Optional<CSSPixelFraction> m_natural_aspect_ratio;
 
+    Vector<JS::NonnullGCPtr<Node>> m_contained_children;
     Vector<JS::NonnullGCPtr<Node>> m_contained_abspos_children;
```

Comment:
m_contained_abspos_children is subset of m_contained_children, so let's leave only m_contained_children and filter out non-abspos elements if needed.

Reply:
One issue I've been running into is that it appears that `m_contained_abspos_children` is not necessarily a subset of `m_contained_children`.

`m_contained_children` are the elements for which this block is the containing block. Whereas `m_contained_abspos_children` are the elements for which this is the closest box that establishes a formatting context. Sometimes the elements in `m_contained_abspos_children` don't have this box as their containing block.

So, it's not clear to me how we would subsume `m_contained_abspos_children` into `m_contained_children`.